macros/CSS_Ref.ejs

```diff
@@ -159,10 +161,17 @@ if (types.some(function(type) { return type === "syntaxes"; })) {
             });
         }
 
-        // Add items within at-rule syntax
-        var items = getItemsFromSyntax(data.syntaxes[syntax]);
+        // Add items within at-rule syntax (e.g. "@annotation")
+        var items;
+        if (syntaxes[Object.keys(syntaxes)[0]].syntax) {
```

Comment:
Using `Object.keys(syntaxes)[0]` for every check is too expensive. Would be better `syntaxes[syntax].syntax` or `'syntax' in syntaxes[syntax]`.
As alternative
```js
var items = getItemsFromSyntax(syntaxes[syntax].syntax || syntaxes[syntax]);
```